pkg/cli/admin/release/info.go

```diff
@@ -1987,6 +2035,306 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
+	}
+
+	ref, err := imagereference.Parse(targetTag.From.Name)
+	if err != nil {
+		return err
+	}
+
+	// not sure this can ever happen, but just sanity-check anyway
+	if ref.ID == "" {
+		return fmt.Errorf("image not digest-based: %s", ref)
+	}
+
+	dbpath := filepath.Join(dir, ref.ID)
+	if _, err = os.Stat(dbpath); os.IsNotExist(err) {
+		err = o.downloadRpmdb(ref, dbpath)
+	}
+	if err != nil {
+		return err
+	}
+
+	db, err := loadRpmdb(dbpath)
+	if err != nil {
+		return err
+	}
+
+	if format == "json" {
+		data, err := json.MarshalIndent(db, "", "  ")
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(o.Out, string(data))
+	} else {
+		pkgs := make([]string, 0, len(db))
+		for pkg := range db {
+			pkgs = append(pkgs, pkg)
+		}
+		sort.Strings(pkgs)
+		fmt.Fprintf(o.Out, "Package contents:\n")
+		for _, pkg := range pkgs {
+			fmt.Fprintf(o.Out, "  %s-%s\n", pkg, db[pkg])
+		}
+	}
+
+	return nil
+}
+
+func (o *InfoOptions) describeRpmDiff(releaseInfo *ReleaseInfo, diff *ReleaseDiff, dir, format, targetImage string) error {
+	if diff.To.Digest == diff.From.Digest {
+		return fmt.Errorf("releases are identical")
+	}
+
+	if targetImage == "" {
+		// find image for machine-os component
+		if tag, ok := releaseInfo.ComponentTags["machine-os"]; !ok {
+			return fmt.Errorf("component machine-os not found in release")
+		} else {
+			targetImage = tag
+		}
+	}
+
+	osDiff, ok := diff.ChangedImages[targetImage]
+	if !ok {
+		// image didn't change; we're done!
+		if format == "json" {
+			fmt.Println("{}")
+		}
+		return nil
+	}
+
+	var err error
+	var fromRef, toRef imagereference.DockerImageReference
+	if fromRef, err = imagereference.Parse(osDiff.From.From.Name); err != nil {
+		return err
+	}
+	if toRef, err = imagereference.Parse(osDiff.To.From.Name); err != nil {
+		return err
+	}
+
+	// not sure this can ever happen, but just sanity-check anyway
+	if fromRef.ID == "" {
+		return fmt.Errorf("image not digest-based: %s", fromRef)
+	}
+	if toRef.ID == "" {
+		return fmt.Errorf("image not digest-based: %s", toRef)
+	}
+
+	fromDbPath := filepath.Join(dir, fromRef.ID)
+	if _, err = os.Stat(fromDbPath); os.IsNotExist(err) {
+		err = o.downloadRpmdb(fromRef, fromDbPath)
+	}
+	if err != nil {
+		return err
+	}
+
+	toDbPath := filepath.Join(dir, toRef.ID)
+	if _, err := os.Stat(toDbPath); os.IsNotExist(err) {
+		err = o.downloadRpmdb(toRef, toDbPath)
+	}
+	if err != nil {
+		return err
+	}
+
+	fromDb, err := loadRpmdb(fromDbPath)
+	if err != nil {
+		return err
+	}
+
+	toDb, err := loadRpmdb(toDbPath)
+	if err != nil {
+		return err
+	}
+
+	removed := make(map[string]string)
+	changed := make(map[string]RpmChangedDiff)
+	for pkg, v := range fromDb {
+		if newVersion, ok := toDb[pkg]; ok {
+			if v != newVersion {
+				changed[pkg] = RpmChangedDiff{
+					Old: v,
+					New: newVersion,
+				}
+			}
+			delete(toDb, pkg)
+		} else {
+			removed[pkg] = v
+		}
+	}
+	rpmDiff := RpmDiff{
+		Changed: changed,
+		Added:   toDb,
+		Removed: removed,
```

Comment:
nit: you might be able to save yourself a bit of space by shifting this up, and avoiding the unattached local variables.  Something like:

```golang
rpmDiff := RpmDiff{
	Changed: make(map[string]RpmChangedDiff),
	Added:   toDb,
	Removed: make(map[string]string),
}
... rpmDiff.Changed[pkg] = RpmChangedDiff{...}
```